src/layer/S2L2ALayer.ts

```diff
@@ -17,6 +17,12 @@ export class S2L2ALayer extends AbstractSentinelHubV3WithCCLayer {
         target: tile.dataUri,
         type: LinkType.AWS,
       },
+      {
```

Comment:
It would be nice if there was a comment here:
```
// S-2 L2A doesn't have previews, but we can use corresponding L1C ones instead:
```

Fftr.

Reply:
I have added it.